coordinator/core/src/main/kotlin/net/consensys/zkevm/ethereum/coordination/conflation/GlobalBlobAwareConflationCalculator.kt

```diff
@@ -32,10 +35,93 @@ class GlobalBlobAwareConflationCalculator(
   override val lastBlockNumber: ULong
     get() = conflationCalculator.lastBlockNumber
 
+  private val gasUsedInBlobHistogram = metricsFacade.createHistogram(
+    category = LineaMetricsCategory.BLOB,
+    name = "gas.used",
+    description = "Total gas used in each blob"
+  )
```

Comment:
This metric's name is misleading IMO "gas used" suggests gas used to send blobs to L1. 

I think we want to measure how much "linea gas is compressed in a blob"; 

So my suggestions are:
- `gas` (without used) - my favourite
- `gas.compressed`